packages/frontend/src/components/send/components/InputAccountId.js

```diff
@@ -179,6 +180,10 @@ class InputAccountId extends Component {
             }
             setAccountIdIsValid(false);
         }
+
+
+        const hapiStatus =  await HapiService.checkAddress({accountId});
+        if (hapiStatus) console.log('hapiStatus', hapiStatus);
```

Comment:
This is effectively just some debugging code, let's please remove it from this PR since it doesn't need to be run in production environments yet. It should get a real implementation once the integration is ready, ideally one that uses our feature flag implementation so that it can be enabled in testnet first.

Reply:
I wrote it mostly for demonstration purposes. Thought that this PR will be a draft for some time, till the full requirements won't be known. But for now, this part just deleted.